### ISSUE TITLE
Creating a new repo is difficult with a screen reader

### ISSUE THREAD

**Issue by GeorgeKerscher** [GeorgeKerscher/Hello-World#2]
In the documentation, it says that every GitHub page has a link to "create something new" but with NVDA I cannot find these links.

**Comment by GeorgeKerscher** [comment]
The problem is that you must maximize the browser to see the link to create a new repo.
